### DISCUSSION TITLE
Accidentally deleted project admin. Now it has no admin.

### DISCUSSION THREAD

**Discussion by MatthewDiPietroTR** [community/community#48298] [Projects and Issues]
### Select Topic Area

Bug

### Body

I accidentally removed myself as the sole admin from a team project I created. Now the project cannot be edited, cannot be removed, and cannot be restored. Ideally, the application wouldn't allow the sole admin to be removed from a given project.

**Comment by valentinojosh** [comment]
Hi!
You could try contacting GitHub support in an attempt to regain access: https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket

Generally speaking its safe practice to keep at least two admins on a team repo. It's definitely a frustrating experience though when accidents like this happen!

**Comment by palapapa** [comment]
GitHub should definitely prevent that from happening. I just suffered the same problem.
